x-pack/plugins/security_solution/public/sourcerer/experimental/use_unstable_security_solution_data_view.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { type SourcererScopeName, type SelectedDataView } from '../store/model';
+
+/**
+ * FOR INTERNAL USE ONLY
+ * This hook provides data for experimental Sourcerer replacement in Security Solution.
+ * Do not use in client code as the API will change frequently.
```

Comment:
I would advise creating a long term feature branch to keep this and not merge it in `main` if it is possible. Would like to understand more about it.

Reply:
I would like to split the work into multiple smaller pull requests, to make it easier to manage on my peers.

Reply:
Okay I understand and that makes sense. But even in that case, in my opinion, the smallest PR that we are merging should be working code. 


Since currently below code doesn't do anything, do you think adding first version of the implementation that we have in mind, would make the PR too big? 

I guess it would make sense to have one working implementation for first version of PR

Let me know what you think.

Reply:
I think it will be too big :)

Reply:
more than 20 files, thats for sure.